doc/nrf/releases/release-notes-0.3.0.rst

```diff
@@ -112,7 +112,7 @@ nRF9160
     This library helps creating an AT command socket and forwards requests and responses from and to the modem.
   * :ref:`lib_nrf_cloud`:
     This library implements features to connect and send data to nRF Cloud services.
-  * **bsdlib**:
+  * **nrf_modem**:
```

Comment:
@umapraseeda thinks so

Reply:
I meant to update the links (like nrfxlib:bsdlib to ref:`nrfxlib:nrf_modem`). May be retain this one and the other bsdlib headings in old release notes, @ru-fu ?

Reply:
Yes, that's how we've done it for the SD Controller as well. Update the references in the release notes (because otherwise, the build will break), but leave the old text as it is.